third_party/terraform/tests/resource_bigquery_table_test.go

```diff
@@ -1048,6 +1072,37 @@ func testAccBigQueryTableFromSheet(context map[string]interface{}) string {
 `, context)
 }
 
+func testAccBigQueryTableFromBigtable(context map[string]interface{}) string {
+	return Nprintf(`
+	resource "google_bigquery_table" "table" {
+		dataset_id = google_bigquery_dataset.dataset.dataset_id
+		table_id   = "tf_test_bigtable_%{random_suffix}"
+
+		external_data_configuration {
+		  autodetect            = true
+		  source_format         = "BIGTABLE"
+		  ignore_unknown_values = true
+
+		  source_uris = [
+			"https://googleapis.com/bigtable/projects/project_id/instances/instance_id/tables/table_name",
```

Comment:
Looks like this is causing an error during testing. This is an invalid source, maybe try using one of the source_uris from another test?

Reply:
I'll try to create a bigtable in the test & generate the URI.

Reply:
@slevenick that's added but I'm unable to run the tests - can you please try again?